packages/realm-server/migrations/1729595943510_billing-adjustments.js

```diff
@@ -0,0 +1,68 @@
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+  pgm.renameTable('credit_balance_events', 'credits_ledger');
+
+  pgm.addColumn('stripe_events', {
+    is_processed: { type: 'boolean', notNull: true, default: false },
+  });
+
+  pgm.alterColumn('subscriptions', 'ended_at', {
+    type: 'timestamp',
```

Comment:
Given the below is this also meant to be `bigint`?

Reply:
confirm, we should treat dates consistently, not as numbers sometimes and date objects other times